Dockerfile

```diff
@@ -26,6 +16,11 @@ WORKDIR /blockd
 COPY ./internal/app/blockd/proto ./proto
 RUN protoc -I/usr/local/include -I./proto --go_out=plugins=grpc:proto proto/api.proto
 
+FROM scratch AS proto
```

Comment:
image with just generated files to make it easy to export it to the host and also use it in base below